config/webpack.config.js

```diff
@@ -518,7 +522,7 @@ module.exports = function (
 				new MiniCssExtractPlugin({
 					filename: contentHash ? '[name].[contenthash].css' : '[name].css',
 					chunkFilename: contentHash ? 'chunk.[name].[contenthash].css' : 'chunk.[name].css',
-					ignoreOrder: noSplitCSS
+					ignoreOrder: app.entry ? true : noSplitCSS
```

Comment:
If `app.entry` is string, is this needed? If not, shouldn't we need another flag?
Also, below code can be better if

```suggestion
					ignoreOrder: app.entry || noSplitCSS
```

Reply:
app.entry is not boolean so `||` is not works...

Reply:
If  that's the case, `!!app.entry || noSplitCSS` will work.